dlk/python/dlk/core/operators.py

```diff
@@ -560,14 +560,16 @@ def __init__(self,
                  data: np.ndarray,
                  dimension_format: str = 'NHWC',
                  packed: bool = False,
-                 actual_shape: List[int] = []) -> None:
+                 actual_shape: List[int] = [],
```

Comment:
Looks  `actual_shape` and `transposed_data` is duplicated with `dimension_format` and `data` 🤔   
In generally, less code is better than more code for maintenance and readable, I think,
Could you use `data` and `dimension_format` instead of `actual_shape` and `transposed_data` ? 😄

Reply:
data don't mean transposed data because both data and transposed data may be used.

dimension format is such as `NCHW' (string),
and actual shape is such as [3,3,3,32] (List[int]).

These two are different.

Reply:
Oh, I mistaken 😭 
`dimension_format` is different from `actual_shape`. Both `data` and `transposed_data` is need to compile for FPGA and others. Thanks.

Maybe there are more nice design to handle kernel format, and nice name of property of `transposed_data` , but I don't have idea, now 😭😭😭